controller/src/main/java/io/pravega/controller/store/stream/VersionedTransactionData.java

```diff
@@ -21,12 +21,15 @@
 @AllArgsConstructor
 public class VersionedTransactionData {
```

Comment:
Is an object of this class persisted anywhere? Do we need to worry about version compatibility?

Reply:
No, this is only an in-memory object.